link_shorterner/src/shortener_shortener.erl

```diff
@@ -0,0 +1,53 @@
+-module(shortener_shortener).
+
+-export([short/1, get/1]).
+
+-export([start_link/0, init/1, handle_call/3, handle_cast/2]).
+
+-behaviour(gen_server).
+
+short(Url) ->
+    gen_server:call(?MODULE, {short, Url}).
+
+get(Url) ->
+    gen_server:call(?MODULE, {get, Url}).
+
+start_link() ->
+    gen_server:start_link({local, ?MODULE}, ?MODULE, [], []).
+
+init(_Args) ->
+    Ets = ets:new(links, [set]),
```

Comment:
create a public named ets and use it directly without having to go through a gen server. you are creating an unnecessary bottleneck and adding an unnecessary process